### PR TITLE
Jupyterhub blog

### PR DESCRIPTION
**Pull request by lusoal**
### What does this PR do?

🛑 Please open an issue first to discuss any significant work and flesh out details/direction - we would hate for your time to be wasted.
Consult the [CONTRIBUTING](https://github.com/awslabs/data-on-eks/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guide for submitting pull-requests.

### More

- [ ] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [ ] Mandatory for new blueprints. Yes, I have added a example to support my blueprint PR
- [ ] Mandatory for new blueprints. Yes, I have updated the `website/docs` or `website/blog` section for this feature
- [ ] Yes, I ran `pre-commit run -a` with this PR. Link for installing [pre-commit](https://pre-commit.com/) locally

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: ai-ml/jupyterhub/helm-values/jupyterhub-values-dummy-gpu.yaml [hunk 1]
```diff
@@ -90,11 +90,11 @@ singleuser:
               display_name: "PyTorch 1.13.1 + torch-neuronx"
               default: true
               kubespawner_override:
-                image: 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-inference-neuronx:1.13.1-neuronx-py310-sdk2.13.2-ubuntu20.04
+                image: 054678736585.dkr.ecr.us-east-1.amazonaws.com/jupyterhub-inferentia-pytorch:1
```
Comment (askulkarni2) [inline ai-ml/jupyterhub/helm-values/jupyterhub-values-dummy-gpu.yaml:93]: Lets obfuscate the account ids.

Comment (vara-bonthu) [reply ai-ml/jupyterhub/helm-values/jupyterhub-values-dummy-gpu.yaml:93]: Agree. Push these images to public ECR under `data-on-eks`.

File: ai-ml/jupyterhub/helm-values/jupyterhub-values-dummy-gpu.yaml [hunk 2]
```diff
@@ -90,11 +90,11 @@ singleuser:
...
             tflow2101:
               display_name: "Tensorflow 2.10.1 + tensorflow-neuronx"
               kubespawner_override:
-                image: 763104351884.dkr.ecr.us-west-2.amazonaws.com/tensorflow-inference-neuronx:2.10.1-neuronx-py310-sdk2.13.2-ubuntu20.04
+                image: 054678736585.dkr.ecr.us-east-1.amazonaws.com/jupyterhub-inferentia-tensorflow:latest
```
Comment (vara-bonthu) [inline ai-ml/jupyterhub/helm-values/jupyterhub-values-dummy-gpu.yaml:97]: same as above comment

File: ai-ml/jupyterhub/variables.tf [hunk 3]
```diff
@@ -7,7 +7,7 @@ variable "name" {
 variable "region" {
   description = "Region"
   type        = string
-  default     = "us-west-2"
+  default     = "us-east-1"
```
[the remark this answers is not in the document]
Comment (lusoal) [reply ai-ml/jupyterhub/variables.tf]: done!

**Pull request merged**